desktop-displaymanagers/greetd-tuigreet/autobuild/beyond

```diff
@@ -0,0 +1,5 @@
+abinfo "Generating manpage ..."
+scdoc < "$SRCDIR"/contrib/man/tuigreet-1.scd > "$SRCDIR"/contrib/man/tuigreet.1
```

Comment:
Only as a suggestion, break down long lines with \. Same for the install command below.